### PR TITLE
chore: v4 calculator compatibility

### PR DESCRIPTION
**Pull request by andivido**
Makes the Magento plugin v4 Calculator Ready. Adds the calc conf to the plugin settings and displays the v4 calc on the product page and checkout if the calc conf api url is configured in the merchants plugin settings.

### PR CHECKLIST

- [x] Ensure any new strings have been translated for import
- [x] Import new translations via `integrations-magento2`'s `make script-install-languages` command
- [x] The plugin version (currently as a const in the `Data.php`) has been updated
- [ ] Tests have been added for any new functionality via `integrations-magento2`'s `make test` command

### CODE REVIEW

**Review by andrei-manolache — APPROVED** [review]

**Review by andrei-manolache — APPROVED** [review]

**Review by freddieline — APPROVED** [review]

**Review by USilviuDvdo — APPROVED** [review]

File: view/frontend/templates/head.phtml [hunk 1]
```diff
@@ -4,15 +4,15 @@
...
+    const dividoEnv = '<?php echo $block->getPlatformEnv(); ?>';
+    const dividoBranding = <?php echo $block->getBrandingJson(); ?>;
 </script>
 <script type="text/javascript" src="<?php echo $block->getScriptUrl(); ?>"></script>
 <script>// <![CDATA[
-function waitForElementToDisplay(selector, time) {
+    function waitForElementToDisplay(selector, time) {
         if(document.querySelector(selector)!=null) {
-            __widgetInstance.init();
+            <?= (empty($block->getCalcConfApiUrl())) ? '__widgetInstance' : '__calculator'; ?>.init();
```
[the remark this answers is not in the document]
Comment (andivido) [reply view/frontend/templates/head.phtml]: Doesn't look as cool though 😎 Changed it now

File: Block/Head.php [hunk 2]
```diff
@@ -40,4 +30,24 @@ public function getBrandingJson()
...
     }
+    
+    public function getCalcConfApiUrl()
+    {
+        return $this->helper->getCalcConfApiUrl();
+    }
+
+    public function getDividoKey()
+    {
+        return $this->helper->getDividoKey();
```
[the remark this answers is not in the document]
Comment (andivido) [reply Block/Head.php]: Fair point. Can I still blame @JonnyCarter after 6 years?

[the remark this answers is not in the document]
Comment (freddieline) [reply Block/Head.php]: Haha. You can but then he might bark back 😆

[the remark this answers is not in the document]
Comment (JonnyCarter) [reply Block/Head.php]: Woof

**Pull request merged**